cl/disclosures/migrations/0004_alter_gross_value_method.sql

```diff
@@ -0,0 +1,9 @@
+-- This clearly makes no change to the database.
```

Comment:
This comment is helpful. Can you change the migration names to end with `_noop.sql` and `_noop.py` as well? That'll help know this at a skim of the filenames.